app/forms/hub/coalition_form.rb

```diff
@@ -0,0 +1,18 @@
+module Hub
+  class CoalitionForm < Form
+    include ActiveModel::Model
+
+    attr_accessor :coalition, :name, :states
+
+    def save
+      coalition.name = name
+      form_states = states.split(",").map { |abbr| States.key_for_name(abbr) } # states are comma delimited in a string, i.e. "Ohio,California"
```

Comment:
Trivial nit:

You're using the variable name `abbr`, but it's not a state abbreviation. It's a full state name, so a better form might be `|name| States.key_for_name(name)`.

I suppose even slicker might be

```
['Ohio', 'New York'].map(&(States.method(:key_for_name)))
```

But actually that took me like 15 minutes of experimentation to figure out, so maybe it will be hard for people to read?

Reply:
I wonder if we can set up the tagify implementation to display the full state name but pass the abbreviation to the backend for us so that we don't have to finagle this in the form. I feel like as often as possible the form should pass the "real" values we want to persist into the db.